pam/integration-tests/testdata/tapes/cli/qr_code.tape

```diff
@@ -22,7 +22,7 @@ Show
 Hide
 Type "./pam_authd login socket=${AUTHD_TESTS_CLI_AUTHENTICATE_TESTS_SOCK}"
 Enter
-Sleep 300ms
+Sleep 800ms
```

Comment:
Some of these are needed because indeed we need to be slower at typing, but this one for example is because in the qrcode case we're passing the user name through an env variable, so it implies some longer processing.

But we know what the ~~solution~~ improvement would be, I'm quite tempted to go with a fork of vhs TBH... :cry: